automated/lib/android-test-lib

```diff
@@ -33,8 +33,6 @@ adb_debug_info() {
 }
 
 initialize_adb() {
-    # enable debug
-    set -x
```

Comment:
+1. I checked existing sigle node tradefed test definitions. We run setup.sh(. ./setup.sh) with the same context from run.sh, and LAVA runs run.sh with sh '-ex' enabled by default, means the trace option should be enabled already.